source/getting-started/js-primer.md

```diff
@@ -55,31 +55,67 @@ export default Ember.Component.extend({
 
 ## Object Literal Shorthand
 
-In ES6 some functionality was added that allows the use of terser syntax for object literal properties and methods, along with some enhanced functionality for property names.
+ES6 introduced a new more succinct syntax for object literal properties and methods, along with some enhanced functionality for property names.
+
+### Property Initializer Syntax
 
-### Property intializer syntax
+Properties can now be set on object literals by including the name of the property only without the typical colon and value. This prevents some duplication when the object property is the same as the local variable you intend to set as the value of that property.
 
-Now properties can be set on object literals without by including the name of the property without the typical colon and value if the object property the same as the local variable you intend to set as the value of that property. This helps reduce duplication. 
+For example, this:
+```javascript
+function someFunc(thing1, thing2) {
+  return {
+    thing1: thing1,
+    thing2: thing2
+  };
+}
+```
 
-For example:
+Can be written like this in ES6 using the property initializer syntax:
```

Comment:
Nice! @jaredgalanis what do you think about saying `can be written, in ES6, as:`? I believe the learner wouldn't have lost the context (the new property initializer syntax) by now and so we don't have to remind them.